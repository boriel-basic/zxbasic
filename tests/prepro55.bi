' Conditional evaluation tests
#define NAME 1

#if NAME
ok 'this should appear
#endif

