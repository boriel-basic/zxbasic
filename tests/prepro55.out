#line 1 "prepro55.bi"




ok 
#line 7 "prepro55.bi"

